packages/botonic-core/tests/core-bot/core-bot.test.ts

```diff
@@ -209,29 +184,7 @@ describe('CoreBot', () => {
       is_first_interaction: false,
     })
     expect(botResponse.lastRoutePath).toEqual('')
-    expect(botResponse.response[0]).toBeDefined()
-    expect(botResponse.response[0].request).toEqual({
-      defaultDelay: 0.4,
-      defaultTyping: 0.6,
-      getString: expect.any(Function),
-      input: { data: 'hello', type: 'text' },
-      lastRoutePath: '',
-      params: {},
-      plugins: {},
-      session: {
-        __retries: 0,
-        user: {
-          locale: LOCALE_EN,
-          country: COUNTRY_GB,
-          system_locale: LOCALE_EN,
-        },
-        is_first_interaction: false,
-      },
-      setUserLocale: expect.any(Function),
-      setUserCountry: expect.any(Function),
-      setSystemLocale: expect.any(Function),
-    })
-    expect(botResponse.response[0].actions).toEqual([null, 'Hi user!', null])
+    expect(botResponse.response).toEqual(['Hi user!'])
```

Comment:
I think that you should preserve as well `[null, 'Hi user!', null]` the null actions. Though they will not be rendered, should be there technically

Reply:
applied